test/converters.test.ts

```diff
@@ -235,20 +231,33 @@ test("decimal converter render with six decimals and thousand separators", async
   expect(rendered).toEqual("4.000.000,000000");
 });
 
-test("decimal converter render, six decimals, no decimalSeparator", async () => {
-  const converter = converters.decimal({ decimalPlaces: 6 });
+test("decimal converter render with six decimals, only showing three", async () => {
+  const converter = converters.decimal({ decimalPlaces: 3 });
   const options = {
+    decimalSeparator: ",",
     thousandSeparator: ".",
     renderThousands: true
   };
-  const value = "4.000000";
+  const value = "4000.000000";
+  const rendered = await converter.render(value, options);
+  expect(rendered).toEqual("4.000,000");
+});
+
+test("decimal converter with can't convert thousandSeparator . and no decimalSeparator", async () => {
+  let message = false;
+  const converter = converters.decimal();
+  const options = {
+    thousandSeparator: ".",
+    renderThousands: true
+  };
+  const value = "4.000";
   const processedValue = converter.preprocessRaw(value, options);
-  const converted = await converter.convert(processedValue, options);
-  const rendered = await converter.render(
-    (converted as ConversionValue<any>).value,
-    options
-  );
-  expect(rendered).toEqual("4.000000");
+  try {
```

Comment:
There's a handy feature in jest for checking whether something throws.
```
  expect(() => {
    drinkFlavor('octopus');
  }).toThrow();
```

Reply:
I see you spend time here extracting the error message (though you only test whether it's truthy?). I would forget about testing for the message for now and just check whether it throws.